app/views/admin/searches/show.html.erb

```diff
@@ -0,0 +1,32 @@
+<div class="admin">
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-one-half">
+      <h1 class="govuk-heading-l">Building search results (<%= @buildings.count %>)</h1>
+    </div>
+  </div>
+
+  <%= render(
+    partial: "admin/shared/search_bar",
+    locals: { search_term: @search_term }
+  ) %>
+
+  <% if @buildings.total_pages > 1 %>
```

Comment:
FYI, Rails has `.one?` and `.many?` which can sometimes make things nice (though here I'm not bothered)